embedding/esp8266/js/main.js

```diff
@@ -0,0 +1,4 @@
+function sysloop(ticknow) {
```

Comment:
I'm worrying about the fact that we have different hello-world applications for different platforms. We can improve it later, just to think.